src/main/kotlin/process/pitch/UtauMode1PitchConversion.kt

```diff
@@ -40,3 +41,15 @@ fun pitchFromUtauMode1Track(pitchData: UtauMode1TrackPitchData?, notes: List<Not
     }
     return Pitch(pitchPoints, false).getAbsoluteData(notes)?.let { Pitch(it, true) }
 }
+const val SEMITONE_TO_CENT_RATIO = 100
```

Comment:
I think this number is obvious, not magic number so just use `100` is ok. (You used 100 directed at above code in the same file)